lib/src/button/button.module.ts

```diff
@@ -1,19 +1,19 @@
 import { ButtonComponent } from './button.component';
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
-import { AnchorComponent } from './anchor.component';
+// import { AnchorComponent } from './anchor.component';
```

Comment:
We probably want to uncomment these. This might be why those unit tests were failing actually. I'd be curious to see if importing the Button Module works with this uncommented.

Reply:
oh.... right
oops, I commented it out to test things one at a time, I didn't mean to keep it commented out.

Good catch